apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.Tests/GoogleCloudConsoleFormatterTest.cs

```diff
@@ -152,6 +153,101 @@ public void ConsoleLoggerOptions_OptionChange_IsReloaded()
         Assert.Equal(expectedJsonWithScope, actualJsonWithScope);
     }
 
+    [Fact]
+    public void ConsoleLoggerOptions_EmptyProjectId_TraceInformationIsNotLogged()
+    {
+        // Don't set the TraceGoogleCloudProjectId.
+        var formatter = CreateFormatter(new GoogleCloudConsoleFormatterOptions());
+
+        var logEntry = new LogEntry<string>(LogLevel.Information, "LogCategory", new EventId(1), "test", exception: null, (state, exception) => state);
+        var writer = new StringWriter { NewLine = "\n" };
+
+        formatter.Write(logEntry, scopeProvider: null, writer);
+
+        var expectedJson = "{\"message\":\"test\",\"category\":\"LogCategory\",\"severity\":\"INFO\"}\n";
+        var actualJson = writer.ToString();
+
+        Assert.Equal(expectedJson, actualJson);
+    }
+
+    [Fact]
+    public void Log_NullActivity_TraceInformationIsNotLogged()
+    {
+        // Set the TraceGoogleCloudProjectId.
+        var formatter = CreateFormatter(new GoogleCloudConsoleFormatterOptions { TraceGoogleCloudProjectId = "my-project-id" });
+
+        // Set Activity.Current to null for clarity. (It is already null here as no Activity is created.)
+        Activity.Current = null;
```

Comment:
Let's not set it explicitly, if MS's default behaviour changes, we want to know about it.
```suggestion
        // Since we haven't started an activity, Activity.Current will be null.
```